pkg/cmd/container/create.go

```diff
@@ -509,12 +509,6 @@ func withNerdctlOCIHook(cmd string, args []string) (oci.SpecOpts, error) {
 			Args: crArgs,
 			Env:  os.Environ(),
 		})
-		scArgs := append(args, "startContainer")
-		s.Hooks.CreateRuntime = append(s.Hooks.StartContainer, specs.Hook{
-			Path: cmd,
-			Args: scArgs,
-			Env:  os.Environ(),
-		})
```

Comment:
Seems introduced in https://github.com/containerd/nerdctl/pull/2839/commits
cc @@alegrey91